### PR TITLE
Add November 2022 newsletter

### PR DESCRIPTION
**Pull request by CollierCZ**
----
:books: Documentation preview :books:: https://writethedocs-www--1838.org.readthedocs.build/en/1838/

### CODE REVIEW

File: docs/blog/newsletter-november-2022.rst [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+As you learn how to advocate for your team, think about how to grow it to meet the needs you discovered. This can mean helping people with professional development or figuring out who can be hired and how to get the budget for that. Creating a long-term plan based on what you determine in the first few months can help you place your team needs in a context that will help get them addressed.
+
+----------------------------
+Minimum viable documentation
+----------------------------
+
+Minimum viable documentation is the minimum required information that's usable and helpful to users. For many reasons, it’s seen positively by some and adversely by others. Those who support it say it’s the best place to start if a product lacks any documentation. Additionally, in places where releases are frequent, minimal documentation enable you to write the essential information and then return to the document to update it once things have stabilized. But there are competing views on minimal documentation, and those who work at places without it believe that the "document everything" approach can lead to many issues, including leaning more towards a mindset of "done is better than perfect."
```
Comment (hillaryfraley) [inline docs/blog/newsletter-november-2022.rst]:
```suggestion
Minimum viable documentation is the minimum required information that's usable and helpful to users. For many reasons, it’s seen positively by some and adversely by others. Those who support it say it’s the best place to start if a product lacks any documentation. Additionally, in places where releases are frequent, minimal documentation enables you to write the essential information and then return to the document to update it once things have stabilized. But there are competing views on minimal documentation, and those who work at places without it believe that the "document everything" approach can lead to many issues, including leaning more towards a mindset of "done is better than perfect."
```

File: docs/blog/newsletter-november-2022.rst [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+Proponents of minimal documentation consider that a product should be capable of functioning with little explanation from its authors. They say this provides a quicker turnaround time on new features, enables faster bug fixes, and simplifies developers’ lives by removing tiresome writing duties. Opponents claim that this strategy results in varying quality across products and can lead to knowledge gaps within teams. They argue that all software should be documented since it eliminates uncertainty when using new systems (and makes life easier for users).
+
+If you’re interested in learning more about minimal viable documentation, read this `blog post from Sarah Moir <https://thisisimportant.net/posts/from-nothing-to-something-with-minimum-viable-documentation/>`__. It covers defining such documentation, assessing documentation states, and structuring documentation.
```
Comment (hillaryfraley) [inline docs/blog/newsletter-november-2022.rst]: I noticed this item switches between `minimum` and `minimal` -- I'm not sure if there's a distinction made in the original conversation but it seems like they're being used interchangeably here.

Comment (CollierCZ) [reply docs/blog/newsletter-november-2022.rst]: That's probably just my editing mistake. Thanks for the note!

File: docs/blog/newsletter-november-2022.rst [hunk 3]
```diff
@@ -0,0 +1,131 @@
+.. post:: November 07, 2022
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – November 2022
+#########################################
+
+Hello once again from the newsletter team. We have a lot of things going on in the community right now.
```
Comment (plaindocs) [inline docs/blog/newsletter-november-2022.rst]: This we team / we community feels confusing to me. How about 

```suggestion
Hello once again from the newsletter team. There are many things going on in the community right now.
```

File: docs/blog/newsletter-november-2022.rst [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+I wrote last month about opportunities to be a part of the Write the Docs community, and this month sees more information. A `new Write the Docs Enhancement Proposal <https://github.com/writethedocs/weps/pull/6>`__ has been opened to provide more clarity on Write the Docs teams and what they do. The change also includes Mikey Ariel stepping down as Prague conference chair and starting as Community Board chair. While I'm sad to see an end to an era of conferences very near me, I'm excited for what she'll bring in the newly defined role. In related news, we have the dates for `conferences in 2023 </blog/2023-conference-announcements/>`__ that you'll probably want to save.
```
Comment (plaindocs) [inline docs/blog/newsletter-november-2022.rst]: Strictly speaking, this is about being more involved, not about being part of the community (arguably by reading you're already part, what do you think?

Comment (plaindocs) [reply docs/blog/newsletter-november-2022.rst]: Also arguably, it's not an end to the Prague conferences, just an announcement that one is not happening in 2023.

File: docs/blog/newsletter-november-2022.rst [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+
+Of course, 2022 isn't over yet and there's still the exciting `Write the Docs Australia conference </conf/australia/2022/>`__ to come. The `full schedule </conf/australia/2022/news/announcing-schedule/>`__ has just been announced, so check out all the interesting talks. There will also be workshops led by people from `The Good Docs Project <https://thegooddocsproject.dev/>`__. Be sure to `get your tickets soon </conf/australia/2022/tickets/>`__ as ticket sales close at the end of the month.
```
Comment (plaindocs) [inline docs/blog/newsletter-november-2022.rst:12]: <3

**Review by plaindocs — APPROVED** [review]
Great stuff.

File: docs/blog/newsletter-november-2022.rst (hunk 4 above)
Comment (CollierCZ) [reply docs/blog/newsletter-november-2022.rst]: Good points, both of them.

**Pull request merged**